config/settings.yml

```diff
@@ -321,3 +321,15 @@ search:
   access_key: SEARCH_GOV_ACCESS_KEY
   affiliate: va
   url: https://search.usa.gov/api/v2
+
+# Time of need
+timeOfNneed:
```

Comment:
this should be `time_of_need`. This file uses snake case and this is misspelled.